mir-ci/mir_ci/conftest.py

```diff
@@ -225,3 +225,23 @@ def xdg(request: pytest.FixtureRequest, tmp_path: pathlib.Path) -> Generator:
                     f.write(contents)
             m.setenv(var, str(var_path))
         yield
+
+
+@pytest.fixture(scope="function")
+def env(request: pytest.FixtureRequest) -> Generator:
+    """
+    Set environment variables.
+
+    Use the `env` mark to provide data, e.g.:
+    @pytest.mark.env(ENV_VARIABLE='value')
+    """
+    if request.config.getoption("--deps"):
+        yield
+        return
```

Comment:
Do we need this? We may even want the env var to affect dep installation, I suppose?